src/test_workflow/benchmark_test/benchmark_test_cluster.py

```diff
@@ -157,7 +155,7 @@ def setup_cdk_params(self, config: dict) -> dict:
             "region": config["Constants"]["Region"],
             "suffix": suffix,
             "securityDisabled": str(self.args.insecure).lower(),
-            "adminPassword": 'myStrongPassword123!' if need_strong_password else None,
+            "adminPassword": 'myStrongPassword123!' if password else None,
```

Comment:
Use get_password to get the password here instead of hardcoding?

Reply:
I was also confused about this, but it seems like this specific test is trying to set "myStrongPassword123!" or None depending on the version, instead of "myStrongPassword123!" or "admin". Thus, I had to make the usage of the util function in this way.

Reply:
This should look like `"adminPassword": admin_password`.